docs/web_advanced.rst

```diff
@@ -436,7 +436,7 @@ the keyword-only ``middlewares`` parameter::
 
 Internally, a single :ref:`request handler <aiohttp-web-handler>` is constructed
 by applying the middleware chain to the original handler in reverse order,
-and is called by the :class:`RequestHandler` as a regular *handler*.
+and is called by the :class:`aiohttp.web.RequestHandler` as a regular *handler*.
```

Comment:
Don't forget to keep how it's being rendered, when possible:
```suggestion
and is called by the :class:`~aiohttp.web.RequestHandler` as a regular *handler*.
```

Reply:
forgot what tilde does:)

Reply:
It influences the rendering